src/handlers/payout/action.ts

```diff
@@ -33,6 +34,10 @@ export const handleIssueClosed = async () => {
 
   if (!issue) return;
 
+  const userHasPermission = await checkUserPermissionForRepoAndOrg(payload.sender.login, context);
```

Comment:
I found an issue testing it out. 
- https://github.com/ubiquibot/staging/issues/119#issuecomment-1682941205
This is not implemented correctly because I'm an admin of the organization @wannacfuture please review